search/search_tests_support/test_search_request.hpp

```diff
@@ -37,8 +37,8 @@ class TestSearchRequest
 
 protected:
   TestSearchRequest(TestSearchEngine & engine, string const & query, string const & locale,
-                    Mode mode, m2::RectD const & viewport, SearchParams::TOnStarted onStarted,
-                    SearchParams::TOnResults onResults);
+                    Mode mode, m2::RectD const & viewport, SearchParams::OnStarted onStarted,
+                    SearchParams::OnResults onResults);
```

Comment:
What do you think about passing |onStarted| and |onResults| by const ref?

Reply:
no